src/ripple/rpc/handlers/WalletPropose.cpp

```diff
@@ -31,9 +31,30 @@
 #include <ripple/rpc/handlers/WalletPropose.h>
 #include <ed25519-donna/ed25519.h>
 #include <boost/optional.hpp>
+#include <cmath>
+#include <map>
 
 namespace ripple {
 
+double
+shannon_entropy (std::string const& input)
```

Comment:
Looks like y'all saw my comment before I deleted it. Yeah, I misread the code, sorry for the noise.